grayskull/pypi/pypi.py

```diff
@@ -59,21 +59,36 @@ def __init__(
         self["build"]["script"] = "<{ PYTHON }} -m pip install . -vv"
 
     @staticmethod
-    def _download_sdist_pkg(sdist_url: str, dest: str):
+    def _pkg_name_from_sdist_url(sdist_url: str):
```

Comment:
Add a docstring here explaining what this does. "Pkg name" can be understood as something else (e.g. project name) but you mean the _downloaded_ archive filename.

Reply:
```suggestion
    def _pkg_name_from_sdist_url(sdist_url: str):
        """
        docstring here
        """
```

Reply:
done